src/PythonBindings/setup.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from distutils.core import setup
+
+setup(
+    name='spectre',
+    version='0.1',
```

Comment:
what does version refer to?  If it is the version of spectre then it should be not be hard-coded.

Reply:
It is the version of the Python package. I have put no thought into how to version this yet. Is there a CMake variable I can substitute in here? Or do you have another suggestion?

Reply:
I don’t think we have even decided how to do versioning.. #615

Reply:
I just substitute in the SpECTRE_VERSION here for now. Seems reasonable for the Python bindings to share the main code's version.